app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -41,5 +42,8 @@ public void initializeUI() {
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.menu, menu);
         super.onCreateOptionsMenu(menu, inflater);
+
+        SearchView searchView = (SearchView) menu.findItem(R.id.menu_search);
```

Comment:
This initialization should come before ` super.onCreateOptionsMenu(menu, inflater);` You'd want to finish your own processes before allowing the system's call to do its thing.